csp/utils/csp_watch_profile.py

```diff
@@ -1,8 +1,9 @@
 import argparse
-import requests
 import time
 from datetime import datetime
 
+import requests
```

Comment:
any idea why ruff moved this import

Reply:
https://peps.python.org/pep-0008/#imports

![image](https://github.com/user-attachments/assets/d6dfae5b-846c-4f0b-b462-e5d1004826f5)

I guess because time, datetime are standard library (1) and requests isn't (2).

Reply:
ugh, I dont know why they decided on that, it just makes it harder to find an import youre looking for
It looks like we can control this though, how strict are we looking to be with this PEP